src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(number) {
+    if (arguments[0] !== undefined) {
```

Comment:
bytheway, you can use an alternative it would be
```suggestion
    if (number !== undefined) {
```

Reply:
@SerhiiKirik I agree, it looks better this way and also easier to understand, thank you!)

Reply:
@JamesSunderlanddd your welcome 😉

Reply:
agree